Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,39 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you swear sometimes?"
```

Comment:
```suggestion
    m 3etd "Say, {w=0.2}[player], {w=0.2}do you ever happen to swear?"
```

Reply:
I prefer the first one to be honest. Here's another suggestion.
```suggestion
    m 3etd "Say, {w=0.2}[player], {w=0.2}do you ever swear?"
```